datastore/elastic_search/create.py

```diff
@@ -1,4 +1,4 @@
-from utils import filter_kwargs
+from .utils import filter_kwargs
```

Comment:
W1618 [34mimport missing `from __future__ import absolute_import`[0m

Reply:
Let's use full imports instead of relative imports
`from datastore.elastic_search.utils import ...`

Reply:
Will be fixing all imports in the next PR.